### DISCUSSION TITLE
Addressing MenuFlyoutItem embedded in MenuFlyoutSubItem

### DISCUSSION THREAD

**Discussion by axtens** [unoplatform/uno#14033] [Q&A]
If my MenuFlyoutSubItem has embedded MenuFlyoutItem, how do I address them in C#?

MenuFlyoutSubItem doesn't appear to have an Items collection. If it had one, I could iterate it.

_MUCH LATER_

Seeing as the control is named I can access it directly and make whatever changes are necessary. It'd be nicer to be able to do it like
```cs
            foreach (var mi in menuBar.Items)
            {
                MainPage.HandlePossibleAmpersand(selectedLanguage[mi.Name], mi);

                foreach (var mii in mi.Items)
                {
                    if (selectedLanguage.ContainsKey(mii.Name))
                        MainPage.HandlePossibleAmpersand(selectedLanguage[mii.Name], mii);
                }
            }
```
and be able to iterate through `mii`'s `.Items` but ...

**Comment by jeromelaban** [comment]
`MenuFlyoutSubItem` is supposed to have `Items` property (https://learn.microsoft.com/en-us/uwp/api/windows.ui.xaml.controls.menuflyoutsubitem.items?view=winrt-22621). Which error did you get when tryin to use it?
